packages/parse5-utils/src/index.js

```diff
@@ -281,7 +275,7 @@ function prependToDocument(document, appendedHtml) {
       if (!head) throw new Error('parse5 did not generated a head element');
       const fragment = parse5.parseFragment(appendedHtml);
       for (const node of adapter.getChildNodes(fragment).reverse()) {
-        prepend(head, node);
+        prepend(head, /** @type {ChildNode} */ (node));
```

Comment:
do you still need this? `getChildNodes` should return `ChildNode[]` already iirc

Reply:
Thanks, removed the type cast.